bin/improved-yarn-audit

```diff
@@ -522,32 +583,50 @@ function loadExclusionsFromFileIfPresent() {
   }
 
   if (excludedAdvisories.length > 0) {
-    console.log(`WARNING: Ignoring ${exclusionsFileName} as exclusions were passed in via command line`)
+    console.log(
+      `WARNING: Ignoring ${exclusionsFileName} as exclusions were passed in via command line`
+    )
     return
   }
 
-  let matchedAdvisories = readFileSync(exclusionsFileName)
+  let matchedNpmAdvisories = readFileSync(exclusionsFileName)
+    .toString()
+    .match(/(?<=^(?:\d+,)*)\d+(?=(?:,\d+)*$)/gm)
+
+  let matchedGithubAdvisories = readFileSync(exclusionsFileName)
     .toString()
-    .match(/(?<=^(?:\d+,)*)\d+(?=(?:,\d+)*$)/gm);
+    .match(/^(GHSA-([a-z0-9]{4})-([a-z0-9]{4})-([a-z0-9]{4}))$/gm)
```

Comment:
New Regex to capture GHSAs

Reply:
@hwood-fg I was wondering the same thing - I wasn't able to find any info on it though. I figured this is still an improvement, we can always adjust later if we figure out what the spec is for these IDs